pkg/awsManager/s3Manager.go

```diff
@@ -60,6 +60,12 @@ func DeleteS3Buckets(client clientpkg.Client, s3BucketsToBeDeleted []*string, lo
 
 // CleanS3Instances cleans s3 buckets
 func CleanS3Instances(client clientpkg.Client, logger logr.Logger) error {
+
+	// TODO: the underlying AWS api returns all buckets for the entire account,
```

Comment:
Could you write up a ticket for this instead of a TODO? That way the team will have visibility on it and we can prioritise accordingly

Reply:
Done. This was more of a nice to have so wasnt sure if we wanted a card or not.